content/docs/rfcs/67/README.md

```diff
@@ -0,0 +1,200 @@
+---
+slug: 67
+title: 67/STATUS-WAKU2-USAGE
+name: Status Waku2 Usage
+status: raw
+category: Best Current Practice
+description: Get familiar with all the Waku protocols used by the Status application.
+editor: Aaryamann Challani <aaryamann@status.im>
+contributors: 
+- Jimmy Debe <jimmy@status.im>
+
+---
+
+# Abstract
+
+Status is a chat application which has several features, including, but not limited to -
+- Private 1:1 chats, described by [55/STATUS-1TO1-CHAT](/spec/55)
+- Large scale group chats, described by [56/STATUS-COMMUNITIES](/spec/56)
+
+This specification describes how a Status implementation will make use of the underlying infrastructure, 
+Waku, which is described in [10/WAKU2](/spec/10).
+
+# Background 
+
+The Status application aspires to achieve censorship resistance and incorporates specific privacy features, 
+leveraging the comprehensive set of protocols offered by Waku to enhance these attributes. 
+Waku protocols provide secure communication capbailites over decentralized networks. 
+Once integrated, an application will benifit from privacy preserving, 
+censorship resistance and spam protected communcation. 
+
+Since Status uses a large set of Waku protocols, 
+it is imperative to describe how each are used. 
+
+# Terminology
+
+| Name  | Description |
+| --------------- | --------- |
+| `RELAY`| This refers to the Waku Relay protocol, described in [11/WAKU2-RELAY](/spec/11) |
+| `FILTER` | This refers to the Waku Filter protocol, described in [12/WAKU2-FILTER](/spec/12) |
+| `STORE` | This refers to the Waku Store protocol, described in [13/WAKU2-STORE](/spec/13) |
+| `MESSAGE` | This refers to the Waku Message format, described in [14/WAKU2-MESSAGE](/spec/14) |
+| `LIGHTPUSH` | This refers to the Waku Lightpush protocol, described in [19/WAKU2-LIGHTPUSH](/spec/19) |
+| Discovery | This refers to a peer discovery method used by a Waku node. |
+| `Pubsub Topic` / `Content Topic` | This refers to the routing and filtering of messages within the Waku network, described in [23/WAKU2-TOPICS](/spec/23/) |
+
+### Waku Node:
+
+Software that is configured with a set of Waku protocols.
+A Status client comprises of a Waku node that can be a relay node or
+a non-relay node.
+
+The Waku network is a group of Waku nodes used to create an open access messaging network, 
+as described in [64/WAKU2-NETWORK](/spec/64).
```

Comment:
Since Status does not run on "the" Waku Network, I'm not sure that RFC 64 is relevant here. Instead, they run on semi-private Status-specific static shards, perhaps best described in https://rfc.vac.dev/spec/57/